llvm/lib/Transforms/IPO/GlobalOpt.cpp

```diff
@@ -2001,6 +2016,9 @@ static bool processInternalGlobal(
       if (isa<UndefValue>(GV->getInitializer())) {
         // Change the initial value here.
         GV->setInitializer(SOVConstant);
+        // If the buid is targeted on Repo, change the GV digest value.
```

Comment:
This should probably say "if the build targets the repo".

Reply:
Fixed, thanks.